app/assets/stylesheets/playtime/wishlist-item.scss

```diff
@@ -0,0 +1,8 @@
+// Styles for wishlist item view
+.wishlist-item {
+  .staff-message,
+  .item-details,
+  .wishlist-item-tools {
```

Comment:
Is this used anywhere?

Reply:
Yes, I found this class here: https://github.com/UJKhokhar/playtime/blob/0438df504dd43b5e64815b76715bf11cfb8c6b9c/app/views/wishlist_items/_wishlist_item.html.erb#L62